### PR TITLE
expose java Kafka producer metrics

### PR DESCRIPTION
**Pull request by dfakhritdinov**

### CODE REVIEW

File: persistence/src/main/scala/akka/persistence/kafka/journal/JournalAdapter.scala [hunk 1]
```diff
@@ -68,7 +68,8 @@ object JournalAdapter {
         val clientId = clientIdOf(config.journal.kafka.producer.common)
         metrics(clientId)
       }
-      KafkaProducerOf[F](producerMetrics)
+      val prefix = clientIdOf(config.journal.kafka.consumer.common)
```
Comment (migesok) [inline persistence/src/main/scala/akka/persistence/kafka/journal/JournalAdapter.scala:71]: client ID is already included in the generated labels

File: persistence/src/main/scala/akka/persistence/kafka/journal/JournalAdapter.scala [hunk 2]
```diff
@@ -68,7 +68,8 @@ object JournalAdapter {
...
+      KafkaProducerOf.apply1[F](prefix, producerMetrics)
```
Comment (migesok) [inline persistence/src/main/scala/akka/persistence/kafka/journal/JournalAdapter.scala:72]: there should be a way to disable producer metrics without disabling overall journal metrics

Comment (dfakhritdinov) [reply persistence/src/main/scala/akka/persistence/kafka/journal/JournalAdapter.scala:72]: why should the metrics be so configurable? I see not much sense in it

File: persistence/src/main/scala/akka/persistence/kafka/journal/JournalAdapter.scala (hunk 1 above)
Comment (dfakhritdinov) [reply persistence/src/main/scala/akka/persistence/kafka/journal/JournalAdapter.scala:71]: in what kind of labels the ID already included?

**Comment by dfakhritdinov** [comment]
The PR must be re-implemented after https://github.com/evolution-gaming/skafka/pull/430 will be merged. New `skafka` feature will allow much smoother integration

File: persistence/src/main/scala/akka/persistence/kafka/journal/JournalAdapter.scala (hunk 1 above)
Comment (migesok) [reply persistence/src/main/scala/akka/persistence/kafka/journal/JournalAdapter.scala:71]: Please try to run it locally - client_id is already included in the labels generated by the collector.
I suspect if you generate the prefix this way, it won't even mention that those are "kafka producer" metrics.

File: persistence/src/main/scala/akka/persistence/kafka/journal/JournalAdapter.scala (hunk 2 above)
Comment (migesok) [reply persistence/src/main/scala/akka/persistence/kafka/journal/JournalAdapter.scala:72]: If they name clash with other things, it's good to be able to disable them.

**Pull request closed without merging**